python/dgl/nn/pytorch/conv/cugraph_relgraphconv.py

```diff
@@ -309,57 +163,58 @@ def forward(self, g, feat, etypes, norm=None):
             so any input of other integer types will be casted into int32,
             thus introducing some overhead. Pass in int32 tensors directly
             for best performance.
-        norm : torch.Tensor, optional
-            A 1D tensor of edge norm value.  Shape: :math:`(|E|,)`.
+        max_in_degree : int, optional
+            Maximum in-degree of destination nodes. It is only effective when
+            :attr:`g` is a :class:`DGLBlock`, i.e., bipartite graph. When
```

Comment:
Is this still only valid for `DGLBlock`?

Reply:
Yes